Makefile

```diff
@@ -108,6 +108,18 @@ compile-requirements: ## Re-compile *.in requirements to *.txt
 	sed '/^[dD]jango==/d' requirements/edx/testing.txt > requirements/edx/testing.tmp
 	mv requirements/edx/testing.tmp requirements/edx/testing.txt
 
+	sed -i '/^[dD]jango-cookies-samesite==/d' requirements/edx/django.txt
```

Comment:
Remove `django-cookies-samesite` from all base, development and testing and move this into django and django30.

Reply:
It looks like I accidentally broke that over 3 years ago: https://github.com/edx/configuration/pull/4508/files#diff-e0912d39d78f6572319b56e288cd622ef87b08f3b6d9ae846c058f40dc365576L1456 .  `django_requirements_file` should be added back to `edxapp_requirements_files`.